core/tracker/tracker2/metrics.go

```diff
@@ -0,0 +1,59 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package tracker2
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/obolnetwork/charon/app/promauto"
+)
+
+var (
+	participationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
can create duplicate metrics, suggest only moving metrics here once they are "enabled" in v2.

Reply:
Added `tracker2` in metrics naming to avoid duplication, v2 will be behind a featureflag until we are confident. These metrics were needed since I copy pasted most of the code and metrics was also part of it.
Will address this in my next PR to wire and add tests.